zebra-chain/src/transaction/tests.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Empty?

Reply:
Yes, it's just a stub since until #124 is done it's difficult to write proptests, but I wanted to add the file as a marker for "tests should be here", "eventually"